code/modules/projectiles/laser.dm

```diff
@@ -281,6 +281,9 @@ toxic - poisons
 	color_green = 0.5
 	color_blue = 0.7
 
+/datum/projectile/laser/pred/cheap // For loot crate pred rifle
+	cost = 25
+
```

Comment:
move cost adjust to parent

Reply:
Oops forgot to commit my removal of that my bad, done.